package.json

```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "babel-core": "5.8.38",
     "immutable": "3.7.6",
-    "jscodeshift": "0.3.20"
+    "jscodeshift": "^0.3.30"
```

Comment:
This is yarn's default behavior. I think the lock file already does that, but maybe it's good to keep it locked down for npm users?